src/math/operations/truncateOps.ts

```diff
@@ -257,11 +259,131 @@ const ambos = makeTruncateTrio({
   },
 })
 
+function getDirection(v1: Vec3D, v2: Vec3D) {
+  return v2.sub(v1).getNormalized()
+}
+
+const augTruncate = makeOpPair({
+  graph: Composite.query
+    .where((s) => {
+      const source = s.data.source
+      return (
+        s.isAugmented() &&
+        !s.isDiminished() &&
+        source.isClassical() &&
+        source.isRegular()
+      )
+    })
+    .map((entry) => ({
+      left: entry,
+      right: entry.withData({
+        source: entry.data.source.withData({ operation: "truncate" }),
+      }),
+    })),
+  middle: "right",
+  getPose($, { geom, specs }) {
+    const source = specs.data.source
+    let caps = Cap.getAll(geom)
+    const isTetrahedron =
+      source.isClassical() && source.isTetrahedral() && source.isRegular()
+    // If source is a tetrahedron, take only the first cap (the other is the base)
+    if (isTetrahedron) {
+      caps = [caps[0]]
+    }
+    const capVertIndices = caps.flatMap((cap) =>
+      cap.innerVertices().map((v) => v.index),
+    )
+    const sourceVerts = geom.vertices.filter(
+      (v) => !capVertIndices.includes(v.index),
+    )
+    const centroid = getCentroid(sourceVerts.map((v) => v.vec))
+    const sourceFaces = geom.faces.filter((f) =>
+      f.vertices.every((v) => !capVertIndices.includes(v.index)),
+    )
+    const scaleFace = sourceFaces.find(
+      (face) => isTetrahedron || face.numSides > 3,
+    )!
+    const cap = caps[0]
+    const mainAxis = cap.normal()
+
+    const boundary = cap.boundary()
+    let crossAxis
+    // FIXME something's wrong if we do this as part of a chain
+    if (specs.isTri()) {
+      const cap1 = caps[1]
+      const cap2 = caps[2]
+      const midpoint = getCentroid([cap1.normal(), cap2.normal()])
+      crossAxis = new Plane(Vec3D.ZERO, boundary.normal()).getProjectedPoint(
```

Comment:
Making sure the axis vectors are orthogonal should just be part of the getPose function (and I think there's another place where this is useful)